examples/get_unchecked_address.rs

```diff
@@ -0,0 +1,48 @@
+//! Get an unchecked from seed. This is just for developing purpose in case you want to see certain address.
+//! For general caes, users should use `get_new_address`.
+//!
+//! Run with:
+//!
+//! ```
+//! cargo run --example get-unchecked-address
+//! ```
+use anyhow::Result;
+use iota::bundle::{Address, TransactionField};
+use iota::crypto::Kerl;
+use iota::signing::{
+    IotaSeed, PrivateKey, PrivateKeyGenerator, PublicKey, Seed, WotsPrivateKeyGeneratorBuilder,
+    WotsSecurityLevel,
+};
+use iota::ternary::{T1B1Buf, TryteBuf};
+use iota_conversion::Trinary;
+
+#[smol_potat::main]
+async fn main() -> Result<()> {
```

Comment:
async runtime is not needed in this case.
```suggestion
fn main() {
```